docs_src/design/ucr/0002-Device-Parent-Child-Relationships.md

```diff
@@ -0,0 +1,115 @@
+If not know or provided by the south-bound Device Service, this information might be added 
+to the Device metadata by the north-bound or analytic services, or by the user.
+
+It is desirable that the means of conveying this information become standardized for those systems
+which provide and use it, hence proposing here that there be a common definition and usage of this metadata.
+
+### Target Users
+- Product Developers
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud User
+- Service Provider
+- Software Integrator
+
+### Description
+Some north-bound protocols and some UI designs present the system devices in a hierarchial manner, 
+where it is necessary to know which devices are parents and which are their children.
+
+These considerations are most important for gateways that are implemented with the EdgeX framework,
+since there are potentially very many south-bound devices connected to a system.
+
+Examples are
+* BACnet - where only one "main" device is present at the point of external connection (eg, UDP port 0xBAC0) and all other devices must be presented as "virtually routed devices" connected to that main "virtual router" device.
+* Azure IoT Hub - where the normal connection for IoT Plug and Play / Digital Twin is for a single device, and any other devices need to somehow fall under that device (eg, with Device Twin "Modules")
+* UI device presentation - where child devices are grouped under their parent, often rolled up until they are expanded to show their data
+* Multi-tenant deployments of multi-point energy meters - where a main meter has up to 80 Branch Circuit Monitoring (BCM) points connected to it, each BCM modeled as a Device consisting of the same 6 or so energy channels (Device Resources), and each BCM is assigned to a particular tenant. Tenants will be given access to the data from their BCM point(s) but not those of other tenants. A gateway may connect more than one of these multi-point energy meters.
+
+Since there are multiple similar uses for this relationship information on the north side, it is proposed to locate
+this relationship metadata in the Device object as accessed from core-metadata by all services, rather than to 
+locate it in each north-bound service (which would be particularly problematic for the UI, which gets its data through REST APIs).
+
+The sound-bound Device Service that creates a Device is ideally the service which establishes this relationship data, though it is possible that it is unaware of the parent-child relationship. It should be permitted, therefore, for this relationship information to also be set by north-bound services (most likely the UI) and simply ignored by the south-bound Device Service.
+
+It is also necessary to indicate which device is the "main" or "publisher" device (ie, the gateway device), 
+as any devices without a configured relationship can be inferred to be children of that device.
```

Comment:
This is a design consideration that should be left to the ADR

Reply:
I've revised the requirements but kept this idea - see if the new approach is acceptable as Requirement.